### DISCUSSION TITLE
Ligninger med 2 variabler giver uendelige løsninger

### DISCUSSION THREAD

**Discussion by stormpmm** [Eduap-com/WordMat#248] [Support]
Jeg prøver at finde to ukendte med to ligninger men solve funktionen giver meget fjollede svar.
Når de to ligninger har ukendte i cos eller sin funktioner giver den resultater som det her:
<img width="1099" alt="Screenshot 2024-10-24 at 15 32 25" src="https://github.com/user-attachments/assets/3a5d37bd-a9c1-4990-9500-48e9bc99ab62">

Hvis man løser det i wolframalpha giver den kun 2 svar som passer med hvad resultatet burde give:
<img width="359" alt="Screenshot 2024-10-24 at 15 33 33" src="https://github.com/user-attachments/assets/fd029f4e-cc01-4600-89b8-33123e805293">

Måske er det rigtigt hvad wordmat siger, men wolframalpha ser ud til at genkende at en vinkel ikke burde være over 360 grader. Er der nogen måde at få wordmat til at gøre ligesom wolframalpha?

**Comment by TorstenOrhoj** [comment]
Jeg prøvede at indsætte begrænsninger på x, y som vist her:
![image](https://github.com/user-attachments/assets/eeff464f-e0c7-4639-9235-69f88e0729cf)

Det giver så følgende:
![image](https://github.com/user-attachments/assets/7a0c4375-0772-45e6-9b3f-6269ab5a6ebe)
dvs. kun 1 løsning.
